### DISCUSSION TITLE
fonts pbf and cors

### DISCUSSION THREAD

**Discussion by sylvain258** [maplibre/maplibre-gl-js#4655] [Q&A]
Hello,

I work with Maplibre, with several vector layers, and among these layers, I have a layer with glyphs. I translate these fonts in pbf (ttf to pbf) and I have this problem "Access to fetch.... from origin ... has blocked by Cors Policy". Cors are enabled on my app, as Maplibre ask it. I have an .htaccess with header :

`Header set Access-Control-Allow-Origin "*"
Header set Access-Control-Allow-Methods "GET,PUT,POST,DELETE,OPTIONS"
Header set Access-Control-Allow-Headers "Content-Type, Access-Control-Allow-Headers, Authorization, X-Requested-With"`

Until yesterday, these fonts worked, but I deleted the fonts directory on my server (mistake from me). I recreated this directory and they are no longer work.

I don't know why it doesn't work.

Thank you for your help

Sylvain BECQUAERT

**Comment by HarelM** [comment]
You'll need to check it by yourself as this is not a maplibre issue.
You can call from the browser console the `fetch` method to one of those files and see what is the problem in the network tab or the console.
You might need to refresh nginx if you added a folder while it was running, IDK...
In any case there's not much we can help...

**Reply by sylvain258** [reply]
Thank you very much for your help. We can close the problem or remove it. There is no point in keeping this problem.
Regards
